docs/getting_started/RuntimeWorkflow.md

```diff
@@ -0,0 +1,85 @@
+## Runtime Workflow
+## Prerequisites
+   Cluster(kind or EKS) should be deployed and available to the nexus runtime deployment
+
+* [Nexus Install](RuntimeWorkflow.md#nexus-install)
+
+* [Nexus Pre-req verify](RuntimeWorkflow.md#nexus-pre-req-verify)
+
+* [Setup Environment](RuntimeWorkflow.md#setup-environment)
+
+* [Install Nexus Admin Runtime](RuntimeWorkflow.md#install-nexus-admin-runtime)
+  
+* [Install Nexus Tenant Runtime](RuntimeWorkflow.md#install-nexus-tenant-runtime)
+
+## Nexus Install
```

Comment:
this is repeated, should we create new file with install and link in required places?

Reply:
Pre-req Verify could also be put there

Reply:
I tried for a common file and link it, but not able to get it while I was doing it in GitLab. will explore it

Reply:
@mkczuk Will track this change with this ticket : https://jira.eng.vmware.com/browse/NPT-641